docker-and-local-requirements.txt

```diff
@@ -0,0 +1,37 @@
+beautifulsoup4==4.9.0
+certifi==2020.4.5.1
+chardet==3.0.4
+click==7.1.2
+cssselect==1.1.0
+Flask==1.1.2
+Flask-API==2.0
+idna==2.9
+itsdangerous==1.1.0
+Jinja2==2.11.2
+joblib==0.15.1
+lxml==4.5.1
+MarkupSafe==1.1.1
+mypy==0.770
+mypy-extensions==0.4.3
+nltk==3.5
+numpy==1.18.4
+pandas==1.0.3
+Pillow==7.1.2
+protobuf==3.12.0
+pyquery==1.4.1
+python-dateutil==2.8.1
+pytz==2020.1
+regex==2019.11.1
+requests==2.23.0
+six==1.14.0
+soupsieve==2.0.1
+sportsreference==0.5.2
+stanza==1.0.1
+torch==1.4.0
+torchvision==0.5.0
+tqdm==4.46.0
+typed-ast==1.4.1
+typing-extensions==3.7.4.2
+urllib3==1.25.9
+Werkzeug==1.0.1
+waitress==1.4.3
```

Comment:
how does `docker-and-local-requirements.txt` differ from `requirements.txt`? whats the reason for needing two? seems kind of tricky to maintain. is it possible to extend one from the other?

Reply:
so the difference is 

`https://download.pytorch.org/whl/cpu/torch-1.4.0%2Bcpu-cp37-cp37m-linux_x86_64.whl`
instead of
`torch==1.4.0
torchvision==0.5.0`

Reason were explicitly referencing the .whl file in AWS is this build is significantly smaller as this version of torch doesn't all for using GPUs. 

I was running into issues with using this requirements.txt file for local and docker builds so this was an attempt to maintain local and docker stability while also wanting to push these out. 

Agree that this is not ideal, will have to look into as having 2 sets of versioning will lead to issues and unnecessary headches